src/app.module.ts

```diff
@@ -1,12 +1,14 @@
-import { Module } from '@nestjs/common';
-import { AppService } from './app.service';
-import { OpenaiService } from './openai/openai.service';
-import { CandidateScanningService } from './candidate-analysis/candidate-scanning.service';
-import { CandidateScanningController } from './candidate-analysis/candidate-scanning.controller';
-import { OpenaiModule } from './openai/openai.module';
+import { Module } from "@nestjs/common";
+import { ConfigModule } from "@nestjs/config";
+
+import { AppService } from "./app.service";
+import { OpenaiService } from "./openai/openai.service";
+import { CandidateScanningService } from "./candidate-analysis/candidate-scanning.service";
+import { CandidateScanningController } from "./candidate-analysis/candidate-scanning.controller";
+import { OpenaiModule } from "./openai/openai.module";
 
 @Module({
-  imports: [OpenaiModule],
+  imports: [ConfigModule.forRoot({ envFilePath: "config.env" }), OpenaiModule],
```

Comment:
I had to add this because with `yarn start` nor `yarn start:debug` no `.env` file was being loaded.